### DISCUSSION TITLE
Update request results in #404 even though record is present.

### DISCUSSION THREAD

**Discussion by AaronHessler** [pocketbase/pocketbase#2785] [Q&A] [answered]
Im currently using svelte kit with pocketbase as my backend and when I try to update the user I get "not found". (See below)
![image](https://github.com/pocketbase/pocketbase/assets/122982679/e5e9005a-a19f-4084-bbc4-47fe492ff304)
But when accessing the api endpoint the record is present.
![image](https://github.com/pocketbase/pocketbase/assets/122982679/4a3cda8e-6ec1-45ef-a966-8747a836ffe0)

Here is the (typescript) code:

`
		const data = {
    		"displayname": displayname,
		};

		await pb.collection('users').update(pb.authStore.model!.id, data);

`

**Comment by ganigeorgiev** [comment, marked as answer]
Make sure that the client request satisfy the `users` collection Update API rule.

_We return 404 in case the client request doesn't satisfy the View, Update, Delete collection API rules, no matter of the authentication state._

**Reply by AaronHessler** [reply]
Thank you!
